### ISSUE TITLE
Dimensional Shard Ore generation in config does nothing

### ISSUE THREAD

**Issue by TheQKnight** [McJty/RFToolsDimensions#213]
Issue description: 

The options in the config to control the generation of Dimensional shard ore do not work. This is very game breaking for my modpack, since players should only be able to craft it.

Steps to reproduce: 

Change all values for Dimensional Shard Ore generation to 0 to stop it spawning in the nether.
Doesn't do anything.

Versions:
* Minecraft: 1.12.2
* Forge: 2655
* CompatLayer (only if on Minecraft 1.10 or 1.11): 0.3.0
* McJtyLib: 2.6.7
* RFTools: 7.33
* RFTools Dimensions: 5.53

Possibly incompatible mods: None

**Comment by josephcsible** [comment]
How do you have CompatLayer installed if you're on 1.12.2?

**Comment by McJty** [comment]
Show me your config

**Comment by TheQKnight** [comment]
Compatlayer 1.11 is compatible with 1.12.2. On twitch launcher it lets you install it like any other mod.

Here's the config.
[CFG.zip](https://github.com/McJty/RFToolsDimensions/files/1963059/CFG.zip)

**Comment by McJty** [comment]
I need the rftools config. That's where dimensional shards are configured. Not in dimensions.cfg

**Comment by TheQKnight** [comment]
There is an option to modify the generation of Dimensional Shard ore in dimensions.cfg

**Comment by TheQKnight** [comment]
I know that there is an option to change it in RFtools, but the bug is that it shows up in the other dimensions config and doesn't work

**Comment by McJty** [comment]
The options for this in dimensions.cfg are unused. Just ignore them

**Comment by TheQKnight** [comment]
Then they shouldn't be there? lol

**Comment by McJty** [comment]
Nope, just ignore them. I forgot to remove them. But they are unused

**Comment by TheQKnight** [comment]
Well, there goes 3 hours of my life trying to figure out why I couldn't get it to stop spawning dimensional ores... Welp
